tests/test_model_v2.py

```diff
@@ -349,6 +349,9 @@ def test_edmx_complex_types(schema):
     assert str(real_prop) == 'StructTypeProperty(Real)'
     assert str(real_prop.struct_type) == 'ComplexType(ComplexNumber)'
 
+    #after correct parsing, new complex type is registered in metadata schema
+    assert str(schema.typ('ComplexNumber')) == 'ComplexType(ComplexNumber)'
```

Comment:
Added before I start splitting Collations for Complex Types as well, this part of Schema class was not tested at all. 

In effect, if you comment out this line, 120 tests fails.

https://github.com/SAP/python-pyodata/blob/21a99b50c3b1718e5b83109d6b4d944d06b7d1e5/pyodata/v2/model.py#L815

But if you comment out this line, 100% tests pass:

https://github.com/SAP/python-pyodata/blob/21a99b50c3b1718e5b83109d6b4d944d06b7d1e5/pyodata/v2/model.py#L827